docs/io/fits/index.rst

```diff
@@ -613,6 +614,10 @@ file opened with update mode:
 
     # closing the file will also flush any changes and prevent further writing
 
+Here we explicitly close the ``hdulist`` opened earlier::
+
+    >>> hdul.close()
+
```

Comment:
Which `hdul` is it closing ? The previous one seems to be closed with the `with` block.

Reply:
I have the same question.

Reply:
I'm looking at it again.

Reply:
I remove this one.  I think it was extraneous.

Reply:
All test pass!